draft-ietf-tls-tls13.md

```diff
@@ -2958,7 +2957,16 @@ the responses).
 
 ### Key and IV Update {#key-update}
 
-     struct {} KeyUpdate;
+     enum { true(1), false(2) (255) } Boolean;
```

Comment:
Ahahahahaha.

Nit: If we're defining this here and not generically, maybe it should be something like:
```
enum { do_not_request_update(0), request_update(1) } KeyUpdateRequest
```
(It doesn't look like you actually need the `(255)`. It implicitly uses enough space for the maximally-defined value otherwise.)

Other nit, I think usually we write `request_update` rather than `requestUpdate`?